cloudferrylib/os/image/glance_image.py

```diff
@@ -307,8 +314,11 @@ def read_info(self, **kwargs):
         for glance_image in self.get_image_list():
             info = self.make_image_info(glance_image, info)
 
+        def get_tags():
+            return {}
```

Comment:
Not sure why this change is needed. Please either remove `get_tags()` method completely, or leave it as is